lib/network/modules/LayoutEngine.js

```diff
@@ -110,6 +110,8 @@ class HierarchicalStatus {
    * @param {String|Number} treeId
    */
   setTreeIndex(node, treeId) {
+    if (treeId === undefined) return;  // Don't bother
```

Comment:
Is this an error condition? Should some warning be elicited?

Reply:
No, it's not. `_crawlNetwork()` can be called for either a full network or a single node. In the latter case, param `tree` in `crawler()` is `undefined` and there's no need to set the tree index.

Would you thing it's better if the test is moved outside of `crawler()`?